### ISSUE TITLE
switching wont work on Firefox 49.0.2

### ISSUE THREAD

**Issue by themaxtech** [canvasowl/ribbbon#27]
Hi the following section i cant switch between users or links
1.  In the clients section i cant change users
2. And the settings menu i cant edit the link and account settings

**Comment by canvasowl** [comment]
Yes this is related to issue #26 I will close that issue for now and keep this thread alive
